src/commandHandlers/help.ts

```diff
@@ -21,4 +21,4 @@ export const description = 'Lists available commands';
 
 export const triggers = ['help'];
 
-export const usage = triggers[0]
\ No newline at end of file
+export const usage = triggers[0]
```

Comment:
This is not a blocker for the PR, just a minor thing. Maybe we could add a ';' here and on `stats.ts` to be consistent with the rest of the code?

Reply:
Oh yes 👍 . The linter isn't working very well. I need to take a look why it is not picking up these things 😢

Reply:
Yes, I was wondering if we could use (or are using but I'm not aware 😅) something like prettier to auto-format and a tool to auto-lint the code before making a commit. Do you run `npm run lint` or did you set it up to be automatic?

Reply:
`npm run lint` will work and CI is running this command, so if we forget to run it locally then CI should pick it up. Something isn't configured right 🤦